### DISCUSSION TITLE
RTP output from srt? (srt-live-transmit)

### DISCUSSION THREAD

**Discussion by dalongra** [Haivision/srt#2238] [Q&A] [answered]
Hello,

I have seen that Haivision Gateway (in AWS) has the option to output TS over RTP (with and without FEC) but I don't find that option in srt-live-transmit tool. 

Is there a way to do it?

Regards,

David

**Comment by ethouris** [comment, marked as answer]
Not with this application. The `srt-live-transmit` tool can only output to SRT, UDP and the standard output. It's a simple demonstration tool only. If you want any more advanced processing, you might want to try ffmpeg, for example.
